File: Mobile/D2/LambdaPP_AndroidApp_D2/app/src/main/java/com/example/lambdapp_androidapp_d2/models/Notification.java
package com.example.lambdapp_androidapp_d2.models;

import java.util.List;

public class Notification
{
    public int not_ID;
    public int emp_ID;
    public int task_ID;
    public String not_Description;
    public String not_Date;
    public List<Task_Equipment_Bridge> task_Equipment_Bridges;
    public List<Task> tasks;
    public boolean not_Viewed;
}

File: Dev/WebApp/LambdaPP_ProjectCore_Alpha/LamdbdaPP_WebAPI_Alpha/Database/Models/Notification.cs
using System.ComponentModel.DataAnnotations;
using System.Text.Json.Serialization;

namespace LambdaPP_WebAPI_Alpha.Database.Models
{
    //This is a model of the "activity" table. It has to match up exactly with the data in the database,
    //in order for the DbContext to work properly!
    //This is what will be returned by the DBContext when it accesses the database.
    //Will need to make models for all other tables we want to access - Gage
    public class Notification
    {
        [Key]
        [JsonPropertyName("not_ID")]
        public int Not_ID { get; set; }
        [JsonPropertyName("emp_ID")]
        public int Emp_ID { get; set; }
        [JsonPropertyName("task_ID")]
        public int Task_ID { get; set; }

        [JsonPropertyName("not_Description")]
        public string? Not_Description { get; set; }

        [JsonPropertyName("not_Date")]
        public DateTime Not_Date { get; set; }

    
        [JsonPropertyName("not_Viewed")]
        public bool Not_Viewed { get; set; }


    }
}
